dbt/manifest/v12.json

```diff
@@ -22702,4 +22724,4 @@
     "unit_tests"
   ],
   "$id": "https://schemas.getdbt.com/dbt/manifest/v12.json"
-}
+}
```

Comment:
the lack of newline may cause issues in dbt-core CI, not 100% one way or the other but may be worth adding it back in